packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -95,6 +95,29 @@ export type PluginExtensionComponentConfig<Props = {}> = {
   extensionPointId: string;
 };
 
+export type PluginExportedComponentConfig<Props = {}> = {
+  /**
+   * The unique identifier of the component
+   * Shoud be in the format of `<pluginId>/<componentName>/<componentVersion>`. e.g. `myorg-todo-app/todo-list/v1`
+   */
+  id: string;
+
+  /**
+   * The title of the component
+   */
+  title: string;
+
+  /**
+   * A short description of the component
+   */
+  description: string;
```

Comment:
Feels a bit odd to make description required, but it is for other extensionConfigs so I guess it should be here too?